### PR TITLE
Feature #r31 errorage #r32 style of lowwer banner

### PR DESCRIPTION
**Pull request by Romanarkko**
## Click Up

* [#R31 Create page "404 not found"](https://app.clickup.com/t/bev4pf)
* [#R32 Implement UI style-> lower banner](https://app.clickup.com/t/bev0u6)

## Summary of issue

No error page, and no handler of errors in a project.
No styles of lowwer banner component. 

## Summary of change

Add error page, and add handler for catchin errors in a project.
Add styles to lowwer banner component. 

## Testing approach

Unit testing.
Manual testing

### CODE REVIEW

**Review by LarysaShashuk — DISMISSED** [review]
Approved

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[1211/221328.466:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
```
Comment (andrew-melnychuk) [inline debug.log]: do we need debug.log file?

Comment (Romanarkko) [reply debug.log]: No, it came by itself. I deleted this file.

**Review by Katy-dev — APPROVED** [review]
good

**Review by andrew-melnychuk — APPROVED** [review]
good

**Review by LarysaShashuk — APPROVED** [review]
approved

**Pull request merged**